applications/dotnix.md

```diff
@@ -0,0 +1,240 @@
+
+###### Previous Web3Foundation grant:
+
+Sebastian Bolaños has applied to a joint grant from the Inter Chain Foundation and the Web3 Foundation for the work on developing WalletConnect V2 during the year 2021.
+
+
+##### Sebastian Kraus
+###### Founder of Sporyon
+
+Sebastian has been an entrepreneur for 6 years and a seasoned CEO. He has experience leading teams and delivering within tight deadlines. Over the past six years, Sebastian has built a broad industry network in the real estate technology sector. Thanks to his dedication and vision, they expanded the agency successfully, acquiring notable clients. Between November 2021 and January 2023, Sebastian enhanced his expertise at Qrucial OÜ, a company specializing in blockchain security. They participated in the Web3 foundation's Grants program, planning and developing the first milestone for QrucialDao on chain auditing solution.
+
+###### Previous Web3Foundation grant:
+
+Sebastian Kraus has applied and delivered the First Milestone of Qrucial DAO Grant.Qrucial OÜ.
+
+
+### Team Code Repos
+
+#### Team Members
+
+- https://github.com/4z3 (tv)
+- https://github.com/sbc64
+- https://github.com/ra33it0
+
+## Development Status 📖
+
+We have started designing the architecture and the interfaces between the different modules. We have kept the first milestone short just to enable quicker iteration of the code and documention.
+
+### Previous research
+-
+## Development Roadmap 🔩
+
+### Overview
+
+- **Total Estimated Duration:** 2 month
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 10000 USD
+
+### Milestone 1: MVP Development and Deployment of NixOS Validator Module
+
+#### Objective: Develop a Minimum Viable Product (MVP) of a validator module deployable as a NixOS service, enabling a simplified one-click deployment process for users.
+
+- **Estimated duration:** 4 weeks
+- **FTE:**  1 FTE
+- **Costs:** 5000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | MIT |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our `Polkadot` nodes and send test transactions, which will demonstrate how the new functionality works. |
```

Comment:
Does "send test transaction" make any sense here? Having read the application I would expect the documentation to heavily focus on the setup of the system and the node itself instead. Documentation on how to interact with a standard Polkadot node already exists, or will you introduce specific features that require additional documentation?

Reply:
Let me clarify with tutorial we mean that we will provide a dotnix quick start guide of how to go from 0 to a running validator that can perform the basic functions up until making a test transaction.
The guide will be focused on how to set up the dotnix validator and other administrative topics.